c/verifythis/duplets.yml

```diff
@@ -0,0 +1,8 @@
+format_version: '1.0'
+
+input_files: 'duplets.c'
+
+properties:
+  - property_file: ../properties/valid-memsafety.prp
```

Comment:
Do you have expected verdicts? If yes, please add (same for other properties and programs, of course).

Reply:
I don't have verdicts. I suspect that there may be some memory errors or the like or bugs in the tasks.

Reply:
Each verdict is true for the time being. Maybe there are some memory errors or similar that I've not taken care of. The intention, however, is that these benchmarks should be completely safe.